app/helpers/application_helper.rb

```diff
@@ -53,6 +53,13 @@ def unread_img
     '/images/bullet_go.png'
   end
 
+  def lastlink_img
```

Comment:
I suspect we'll need a cucumber test for this but I can show you how to make those later